frugalos_segment/src/synchronizer.rs

```diff
@@ -37,6 +37,8 @@ pub struct Synchronizer {
     enqueued_delete: Counter,
     dequeued_repair: Counter,
     dequeued_delete: Counter,
+    repairs_success_total: Counter,
+    repairs_failure_total: Counter,
```

Comment:
Probably it is not necessary to add these metrics to Synchronizer. They are used only in `RepairContent`.

Reply:
This was wrong.